README.md

```diff
@@ -1,112 +1,6 @@
-<h3 align="center">
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin for <a href="https://matplotlib.org/">Matplotlib</a>
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-</h3>
+<h2 align="center">🚧 This repository has moved to <a href="https://github.com/catppuccin/python/tree/main">catppuccin/python</a> 🚧</h2>
 
-<p align="center">
-	<a href="https://github.com/brambozz/matplotlib-catppuccin/stargazers"><img src="https://img.shields.io/github/stars/brambozz/matplotlib-catppuccin?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
-	<a href="https://github.com/brambozz/matplotlib-catppuccin/issues"><img src="https://img.shields.io/github/issues/brambozz/matplotlib-catppuccin?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
-	<a href="https://github.com/brambozz/matplotlib-catppuccin/contributors"><img src="https://img.shields.io/github/contributors/brambozz/matplotlib-catppuccin?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
-</p>
+Follow the examples in the README of the new repo to upgrade your code. No changes required on the calls, only on the imports.
```

Comment:
```suggestion
See https://github.com/catppuccin/python#matplotlib for the documentation and instructions for updating your code.
```

Added a link to where the new Matplotlib section will be once your upstream PR is merged. Also, best not to say "No changes required on the calls, only on the imports." since that may not be the case down the road and once we archive this repository we can no longer edit the message (easily at least).